packages/web-api/README.md

```diff
@@ -394,6 +383,6 @@ If you get stuck, we're here to help. The following are the best ways to get ass
 
   * [Issue Tracker](http://github.com/slackapi/node-slack-sdk/issues) for questions, feature requests, bug reports and
     general discussion related to these packages. Try searching before you create a new issue.
-  * [Email us](mailto:developers@slack.com) in Slack developer support: `developers@slack.com`
-  * [Bot Developers Hangout](https://community.botkit.ai/): a Slack community for developers
-    building all types of bots. You can find the maintainers and users of these packages in **#sdk-node-slack-sdk**.
+  * [Email us](mailto:feedback@slack.com): `feedback@slack.com`
```

Comment:
Should they email feedback@slack.com or support@slack.com as indicted in [other repos](https://github.com/slackapi/bolt-js#getting-help)

Reply:
My understanding is `feedback@slack.com`